pkg/hs/urbit-king/lib/Urbit/Vere/Dawn.hs

```diff
@@ -122,24 +106,25 @@ dawnSendHTTP endpoint requestData = liftIO do
 class RequestMethod m where
   getRequestMethod :: m -> Text
 
-data RawResponse = RawResponse
+data RawResponse r = RawResponse
   { rrId :: Int
-  , rrResult :: Text
+  , rrResult :: r
   }
   deriving (Show)
 
-instance FromJSON RawResponse where
+instance FromJSON r => FromJSON (RawResponse r) where
   parseJSON = withObject "Response" $ \v -> do
-    rrId <- v .: "id"
+    rawId <- v .: "id"
     rrResult <- v .: "result"
+    let rrId = read rawId
```

Comment:
So it changed from `{id: 124, result: "blah"}` to `{id: "123", result: "blah"}`; do I have that right?

Reply:
Correct. (This is due to a temporary limitation in the Urbit-side RPC handling, we might go back to proper integers before too long.)